src/blueapi/service/main.py

```diff
@@ -173,6 +175,34 @@ def delete_submitted_task(
     return TaskResponse(task_id=handler.clear_task(task_id))
 
 
+def validate_task_status(v: str) -> TaskStatusEnum:
+    v_upper = v.upper()
+    if v_upper not in TaskStatusEnum.__members__:
+        raise ValueError("Invalid status query parameter")
+    return TaskStatusEnum(v_upper)
+
+
+@app.get("/tasks/", response_model=TasksListResponse, status_code=status.HTTP_200_OK)
+def get_tasks(
+    task_status: str,
```

Comment:
Should: I'm surprised we didn't already have a `/tasks` endpoint, shall we make the query parameter optional so it provides all tasks if no desired state is supplied?

Reply:
(you could also just make an issue for this if preferred, so I'll retroactively change it to a "could")

Reply:
added this endpoint